lib/logflare/logs/search/logs_search_queries.ex

```diff
@@ -24,9 +36,25 @@ defmodule Logflare.Logs.SearchQueries do
     |> select([t], %{
       timestamp:
         fragment(
-          "(`$$__DEFAULT_DATASET__$$`.LF_TIMESTAMP_TRUNC_WITH_TIMEZONE(?, ?, ?)) as timestamp",
+          """
```

Comment:
can abstract out to avoid duplication.

Reply:
Yea we can remove the function entirely since we do the tz formatting client side.
let's do it in separate PR.